vermouth/tests/test_annotate_mut_mod.py

```diff
@@ -142,6 +153,23 @@ def test_annotate_modifications(example_mol, modifications, mutations, expected_
     for node_idx, mods in expected_mut.items():
         assert _subdict(mods, example_mol.nodes[node_idx])
 
+
+def test_single_residue_mol():
+    mol = Molecule(force_field=ForceField(FF_UNIVERSAL_TEST))
+    nodes = [
+        {'chain': 'A', 'resname': 'A', 'resid': 2},
+        {'chain': 'A', 'resname': 'A', 'resid': 2},
+    ]
+    mol.add_nodes_from(enumerate(nodes))
+    mol.add_edges_from([(0, 1)])
+
+    modification = [({'resname': 'A', 'resid': 2}, 'C-ter'),]
+    annotate_modifications(mol, modification, [])
+
+    assert mol.nodes[0] == {'modification': ['C-ter'], 'resname': 'A', 'resid': 2, 'chain': 'A'}
+    assert mol.nodes[1] == {'modification': ['C-ter'], 'resname': 'A', 'resid': 2, 'chain': 'A'}
+
+
```

Comment:
I'm of the opinion this should raise an error because or a single amino acids C and N terminus are ill-defined terms in the martini sense, aren't they.